esp/src/src-react/components/Title.tsx

```diff
@@ -87,6 +88,34 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
         [setModernMode]
     );
 
+    const [logCount, setLogCount] = React.useState(0);
+    React.useEffect(() => {
+        const errorCodes = [Level.alert, Level.critical, Level.emergency, Level.error];
+        const warningCodes = [Level.warning];
+        const errorAndWarningCodes = errorCodes.concat(warningCodes);
+
+        const logCounts = {
+            errors: log.filter(msg => errorCodes.includes(msg.level)).length,
+            warnings: log.filter(msg => warningCodes.includes(msg.level)).length,
+            other: log.filter(msg => !errorAndWarningCodes.includes(msg.level)).length,
```

Comment:
Minor, but `other = log.length - errors - warnings`?

Reply:
That's right. The CounterBadge only has 3 colors, so errors = danger, warnings = important and anything else = informative